client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -50,12 +54,41 @@ object DiagramView {
               }
             }),
             cls := "bg-gray-900 text-gray-50 rounded my-3 p-3",
+            span(
+              "Hello, "
+              // child.text <-- AppState.dashboardConfig.signal.map(_.diagrams.filter(d => d.id == cfg.id))
```

Comment:
@atooni is this how I hook the AppState?

Reply:
@EstebanMarin, normally I wouldnt expect that we would use `AppState.diagrams`with a filter within the `DiagramView`. We are passing a `Signal[DiagramView]` into the view, so each view already "knows" what config it is linked to. Even though your approach could work, it will add quite some overhead and makes constructing `Update...` commands a bit more difficult. 

I will be on Discord throughout the day, feel free to touch base - then we might find for pairing up.